de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc

```diff
@@ -160,12 +160,255 @@ Congratulations. You have successfully set up your Git integration!
 
 == Generic Systems Engineering (Concept) Language 
 
-=== Categories and Properties
+Concepts are described using the Generic Systems Engineering Language (GSEL).
+The following gives a break-down on the key words and functionalities provided by the GSEL.
+
+=== Concepts
+
+A  _Concept_ is always the root of a concept.concept file.
+It serves as the main container of the concept.
+It also fulfills the task of carrying global meta-information.
+
+A Concept may also have the following attributes:
+
+displayname::
+This name will be shown to the user when interacting with the concept in the UI.
+For example when activating a concept, the displayname is shown.
+
+description::
+The description of the concept.
+
+version:: 
+The version number of the concept. 
+Must be given in the form of `.` separated numbers.
+For each version number, the IDE tools generate a corresponding `concept_vVERSION.xmi` in the concept folder.
+This file is used to automtically migrate old repositories based on an older concept version to the newest version.
+

```

Comment:
Difference name and displayName

Reply:
Added name description and extended description of displayname.